staff/albert-escolar/cv/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Albert Escolar CV</title>
+</head>
+
+<body>
+    <header>
+        <h1>Curriculum Vitae</h1>
+    </header>
+
+    <ul>
+        <li><b>Name:</b> Albert Escolar Cano</li>
```

Comment:
`<b>` is not a semantic tag 😄

Reply:
More info in the following [link](https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em)

Reply:
<b> changed for <strong>, it shoud be ok now